.github/workflows/stale.yml

```diff
@@ -47,5 +46,13 @@ jobs:
           close-issue-message: >
             This issue has been closed because it has not received any activity in the last 14 days
             since being marked as 'stale'
+          # stale PRs
+          stale-pr-label: 'stale'
+          exempt-pr-labels: 'not-stale,security'
+          stale-pr-message: 'This pull request has been marked as stale due to 30 days of inactivity. It will be closed in 1 week if no further activity occurs. If you think that’s incorrect or this pull request requires a review, please simply write any comment. If closed, you can revive the PR at any time and @mention a reviewer or discuss it on the dev@iceberg.apache.org list. Thank you for your contributions.'
+          close-pr-message: 'This pull request has been closed due to lack of activity. This is not a judgement on the merit of the PR in any way. It is just a way of keeping the PR queue manageable. If you think that is incorrect, or the pull request requires review, you can revive the PR at any time.'
+          stale-pr-label: 'stale'
+          days-before-pr-stale: 30
+          days-before-pr-close: 7
           ascending: true
-          operations-per-run: 100
+          operations-per-run: 1000
```

Comment:
Looks like they suggest numbers as high of 1000 for debugging: https://github.com/actions/stale?tab=readme-ov-file#operations-per-run

I believe 100 is already quite substantial and will generate a lot of noise. Raising this to 1k might flood people's mailboxes. WDYT?

Reply:
I set 1000 here for debugging first (to be back on 100 later). I think 100 is decent number (it's the same of the batch, it won't "limit" the issues/PRs considered to 100 actually). Let me back to 100 (debugging is probably not required if it works fine :)).

Reply:
I saw @manuzhang's comment after I went over this. Maybe we should do some manual backlog cleaning. For example, close PRs of Spark/Flink versions that are not supported anymore.

Reply:
My original intention was to do a first run with 1000 and then back to 100 to do a first pass on PR that should be closed. But we can also do this first pass "manually".